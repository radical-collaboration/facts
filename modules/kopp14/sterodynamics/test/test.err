kopp14_fit_oceandynamics.py:183: RuntimeWarning: Mean of empty slice
  ext_num = sZOS[year_2099_idx,:,:] - np.nanmean(sZOS[year_2099_idx,:,:], axis=0)
kopp14_fit_oceandynamics.py:200: RuntimeWarning: Mean of empty slice
  OceanDynMean = np.nanmean(sZOS, axis=1) * 1000
/home/rk509/.conda/envs/conda-entk/lib/python3.7/site-packages/numpy/lib/nanfunctions.py:1671: RuntimeWarning: Degrees of freedom <= 0 for slice.
  keepdims=keepdims)
kopp14_fit_oceandynamics.py:213: RuntimeWarning: Mean of empty slice
  zos_demean = sZOS - np.nanmean(sZOS, axis=1)[:,np.newaxis,:]
kopp14_fit_oceandynamics.py:217: RuntimeWarning: invalid value encountered in true_divide
  OceanDynTECorr = corr_num / corr_denom
