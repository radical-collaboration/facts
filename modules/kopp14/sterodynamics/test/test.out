Initiating test in /scratch/rk509/test.1666725638...

Extracting data files...

Executing workflow...
python kopp14_preprocess_oceandynamics.py --pipeline_id oceandynamics-kopp14-oceandynamics --scenario rcp85 --zostoga_model_dir ./SLR_ALL/ --zos_model_dir ./zosfinal/ --baseyear 2005 --pyear_start 2020 --pyear_end 2100 --pyear_step 10 --locationfile test/location_list.lst
python kopp14_fit_oceandynamics.py --pipeline_id oceandynamics-kopp14-oceandynamics
python kopp14_project_oceandynamics.py --pipeline_id oceandynamics-kopp14-oceandynamics --nsamps 200 --seed 4321
python kopp14_postprocess_oceandynamics.py --pipeline_id oceandynamics-kopp14-oceandynamics --nsamps 200 --seed 4321

Collecting output files...
oceandynamics-kopp14-oceandynamics_globalsl.nc
oceandynamics-kopp14-oceandynamics_localsl.nc

Skipping clean up...
